features/karaf/openhab-core/pom.xml

```diff
@@ -17,7 +17,6 @@
   <properties>
     <jetty.version>9.4.12.v20180830</jetty.version>
     <jna.version>4.5.2</jna.version>
-    <nrjavaserial.version>3.15.0.OH2</nrjavaserial.version>
```

Comment:
That property has not been necessary at all, as it is used at one place -- in the feature.
There is no "win" to define it here and reference in the feature.
We can define it in the feature.

Now the bundle declaration in the feature has been moved at all to another feature repository.